mixins/form.jsx

```diff
@@ -96,7 +95,7 @@ module.exports = {
     this.updateHeight();
   },
   submit: function(action, props, callback) {
-    props.localeCode = this.state.localeCode || "US";
+    props.locale = this.props.locales[0];
```

Comment:
For URL en-US is what you want, but parsing it from locale to what PayPal expect might not be a good idea unless we have list of known one from PayPal and make a dictionary to lookup for that